From: Lubomir Rintel <lkundrak@v3.sk>
Date: Mon, 18 Dec 2017 08:27:25 +0100
Subject: libnm-core/tc-config: remove deprecation guards from typedefs

They're triggered by the prototypes in header file inclusion, even
though no routines or variables that use the type are actually used.

(cherry picked from commit 685cb5c88bf90b285a18a98a6bbeafa559938fee)
(cherry picked from commit 8d120a6fda1761430241367781c57d302ffc5fe3)
---
 libnm-core/nm-setting-tc-config.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/libnm-core/nm-setting-tc-config.h b/libnm-core/nm-setting-tc-config.h
index 736d5ad..8117d5e 100644
--- a/libnm-core/nm-setting-tc-config.h
+++ b/libnm-core/nm-setting-tc-config.h
@@ -28,7 +28,6 @@
 
 G_BEGIN_DECLS
 
-NM_AVAILABLE_IN_1_10_2
 typedef struct NMTCQdisc NMTCQdisc;
 
 NM_AVAILABLE_IN_1_10_2
@@ -61,7 +60,6 @@ void        nm_tc_qdisc_set_handle           (NMTCQdisc *qdisc,
 NM_AVAILABLE_IN_1_10_2
 guint32     nm_tc_qdisc_get_parent           (NMTCQdisc *qdisc);
 
-NM_AVAILABLE_IN_1_10_2
 typedef struct NMTCAction NMTCAction;
 
 NM_AVAILABLE_IN_1_10_2
@@ -96,7 +94,6 @@ void        nm_tc_action_set_attribute       (NMTCAction *action,
                                               const char *name,
                                               GVariant *value);
 
-NM_AVAILABLE_IN_1_10_2
 typedef struct NMTCTfilter NMTCTfilter;
 
 NM_AVAILABLE_IN_1_10_2
